docs/modules/reference/pages/glossary.adoc

```diff
@@ -91,10 +92,10 @@ They let you display data from different devices and sources (SNMP, ICMP, HTTP)
 (See xref:operation:admin/webui/opsboard/dashlet/ksc.adoc#ksc[KSC Reports].)
 
 Kubernetes:: Kubernetes is an open-source container orchestration system for automating software deployments, scaling, and management.
-Originally designed by Google, it is now maintained by the Cloud Native Computing Foundation.
+Originally designed by Google, the Cloud Native Computing Foundation now maintains it.
 (See https://kubernetes.io/[Kubernetes].)
 
-Location:: The geographical location of a node that is being managed using OpenNMS.
+Location:: The monitoring location of a node.
 Nodes can be manually assigned a location (See xref:reference:provisioning/handlers/dns.adoc[Location]), or the location can be determined using the GeoIP Provisioning Adapter (See xref:reference:provisioning/adapters/geoip.adoc[GeoIP]).
```

Comment:
I would still delete this sentence. As Mark mentioned, the DNS handler isn't relevant here. 

Replace it with a sentence that explicitly states there is a difference between location and geolocation.
"This differs from geolocation which is the actual physical location of the node."

Add the term "geolocation" to the glossary, and define it as the physical location of the node. I don't think we need to provide any more detail in the glossary.

Reply:
This was resolved in the most recent commit. :)

Reply:
Oh. Sorry. That's strange. It didn't show up when I did the review. I will look again.